### PR TITLE
ci: adding workflow for pr conventions

### PR DESCRIPTION
**Pull request by hamza-m-masood**
### Which problem does the PR fix?

related: https://github.com/camunda/camunda-platform-helm/issues/1581
### What's in this PR?
Enforcing PR conventions

### Checklist

Please make sure to follow our [Contributing Guide](../blob/main/docs/contributing.md).

**Before opening the PR:**

- [ ] In the repo's root dir, run `make go.update-golden-only`.
- [ ] There is no other open [pull request](../pulls) for the same update/change.
- [ ] Tests for charts are added (if needed).
- [ ] In-repo [documentation](../blob/main/docs/contributing.md#documentation) are updated (if needed).

**After opening the PR:**

- [ ] Did you sign our CLA (Contributor License Agreement)? It will show once you open the PR.
- [ ] Did all checks/tests pass in the PR?

### CODE REVIEW

**Comment by hamza-m-masood** [comment]
Interestingly when I only ahve one commit I get this error:
```
Error: Pull request has only one commit and it's not semantic; this may lead to a non-semantic commit in the base branch (see https://github.com/community/community/discussions/16271). Amend the commit message to match the pull request title, or add another commit.
```

File: .github/workflows/pr-conventions.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: Pull Request Conventions
```
Comment (aabouzaid) [inline .github/workflows/pr-conventions.yaml:1]: Following the same convention let's call it `Repo - Pull Request Conventions`

Comment (hamza-m-masood) [reply .github/workflows/pr-conventions.yaml]: done

**Review by aabouzaid — APPROVED** [review]
LGTM, thank you :+1:

**Pull request merged**